docker/openemr/7.0.1/php.ini

```diff
@@ -1790,12 +1790,12 @@ ldap.max_links = -1
 
 ; When disabled, you must reset the OPcache manually or restart the
 ; webserver for changes to the filesystem to take effect.
-;opcache.validate_timestamps=1
+opcache.validate_timestamps=1
 
 ; How often (in seconds) to check file timestamps for changes to the shared
 ; memory storage allocation. ("1" means validate once per second, but only
 ; once per request. "0" means always validate)
-;opcache.revalidate_freq=2
+opcache.revalidate_freq=60
```

Comment:
Looks like this is the only setting that will be different from default (the other settings above are not necessary since they are already set to that by default: https://www.php.net/manual/en/opcache.configuration.php)

Reply:
Actually I would just keep this setting at default. @adunsulag was having a very good performance improvement with this setting on default of 2. The problem with 60 is that it will kill the developers (ie. imagine needing to wait 60 seconds for a code change to be implemented while developing/testing :) ).

Reply:
We could do php.ini.prod and php.ini.flex and then pick one or the other -- one with a sixty-second clock and one with a two-second clock? Pass in arguments to the build to pick the one or the other. Might save it for later though.

Reply:
Note not too attached to what that setting is since the standard dev environments have xdebug going, so this won't be on anyways. However, rather than changing here, would just change it in the if/else block I noted below.